app/com/yetu/oauth2provider/utils/DateUtility.scala

```diff
@@ -30,45 +41,21 @@ object DateUtility {
 
   // java Date <---> String
 
-  private val LDAP_DATE_FORMAT = "yyyyMMddHHmmss"
-
-  private val ldapDateFormat: SimpleDateFormat = new SimpleDateFormat(LDAP_DATE_FORMAT)
-
   def dateToString(date: Date): String = {
```

Comment:
If you're using this a lot, creating a `new SimpleDateFormat` each time might have a performance impact